runtime/BUILD.bazel

```diff
@@ -30,6 +30,7 @@ go_library(
         "@com_github_golang_protobuf//jsonpb:go_default_library_gen",
         "@com_github_golang_protobuf//proto:go_default_library",
         "@com_github_golang_protobuf//protoc-gen-go/generator:go_default_library_gen",
+        "@com_github_grpc_ecosystem_grpc_gateway//runtime/internal:go_default_library",
```

Comment:
I don't think this should be necessary, I can't imagine we don't already import the internal package somewhere in here and it's worked in the past.